identity-iota/src/client/client_builder.rs

```diff
@@ -1,23 +1,25 @@
 // Copyright 2020-2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::client::Client;
-use crate::client::Network;
-use crate::error::Result;
+use crate::{
+  client::{Client, Network},
+  error::Result,
+};
 
 /// A `ClientBuilder` is used to generated a customized `Client`.
-#[derive(Clone, Debug)]
 pub struct ClientBuilder {
   pub(crate) network: Network,
   pub(crate) nodes: Vec<String>,
+  pub(crate) node_sync_enabled: bool,
 }
 
 impl ClientBuilder {
   /// Creates a new `ClientBuilder`.
-  pub const fn new() -> Self {
+  pub fn new() -> Self {
     Self {
-      network: Network::Mainnet,
+      network: Network::default(),
       nodes: Vec::new(),
+      node_sync_enabled: true,
```

Comment:
can this default to `false` since it seems to be required at the moment?

probably also good to use a constant